pkg/api/authz.go

```diff
@@ -340,3 +339,13 @@ func anonymousPolicyExists(config *config.AccessControlConfig) bool {
 
 	return false
 }
+
+func isExtensionURI(requestURI string) bool {
+	/* Notes:
+	 	- since we only do READ actions in extensions, we can bypass authz for them
```

Comment:
This note is unrelated to the actual functionality provided by the function. Move it somewhere else?